personal_assistant/pa_tag/templates/tags/tags.html

```diff
@@ -0,0 +1,24 @@
+{% extends "pa_core/home.html" %}
```

Comment:
Унаслідуватись від загального шаблону для сторінок із всіма сутностями певного виду і так само назвати свій шаблон для узгодженості назв:
```suggestion
{% extends 'pa_core/overview.html' %}
```